plugins/weather.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+#  weather.rb
+#  Author: William Woodruff
+#  ------------------------
+#  A Cinch plugin that provides APIXU weather interaction for yossarian-bot.
+#  ------------------------
+#  This code is licensed by William Woodruff under the MIT License.
+#  http://opensource.org/licenses/MIT
+
+require 'bundler/setup' # needed for apixu
```

Comment:
the following(weather.rb:11 being where `require "apixu"` is): 
```
Traceback (most recent call last):
	7: from yossarian-bot.rb:36:in `<main>'
	6: from yossarian-bot.rb:36:in `each'
	5: from yossarian-bot.rb:37:in `block in <main>'
	4: from /usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	3: from /usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	2: from /home/cat/gitstuff/yosbot/plugins/weather.rb:11:in `<top (required)>'
	1: from /usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require
```

Reply:
>Add this to your app (required because the library is installed from local path):
>`require 'bundler/setup'`

from the [apixu gem page](https://github.com/apixu/apixu-ruby#gemfile), I'm not sure why it needs it either or why I cant `require "apixu"` without it